control/prometheus.py

```diff
@@ -150,18 +150,30 @@ def _get_gw_metadata(self):
 
     @timer
     def _get_bdev_info(self):
-        with self.gateway_rpc.rpc_lock:
-            return rpc.bdev.bdev_get_bdevs(self.spdk_rpc_client)
+        try:
+            with self.gateway_rpc.rpc_lock:
+                return rpc.bdev.bdev_get_bdevs(self.spdk_rpc_client)
+        except Exception:
+            logger.exception(f"Error trying to call bdev_get_bdevs()")
```

Comment:
nitpick - f strings without variables usually get picked up by linting (flake8) - common across all the logger.exception statements.

Reply:
Done